apps/api/src/app/widgets/widgets.controller.ts

```diff
@@ -266,8 +266,8 @@ export class WidgetsController {
       organizationId: subscriberSession._organizationId,
       subscriberId: subscriberSession.subscriberId,
       environmentId: subscriberSession._environmentId,
-      markAs: 'read',
-      feedIds,
+      markAs: MarkMessagesAsEnum.READ,
+      feedIdentifiers: feedIds,
```

Comment:
Would we like to change the `feedIds` to `feedIdentifier` as a breaking change?

Otherwise, we could duplicate this one as a new endpoint ` @Post('/messages/mark-all')` with `feedIdentifier` body param? and mark the current one as deprecated?

I am raising it as a following to the conversation we have about having naming more consistent.

Same in `markAllUnseenAsSeen`.

Reply:
apologies!!! I forgot about the next steps till I got there 😅